.circleci/config.yml

```diff
@@ -9,6 +9,19 @@ parameters:
         default: false
 
 jobs:
+    # Ensure running with CircleCI/huggingface
+    check_circleci_user:
+        docker:
+            - image: cimg/python:3.7.12
+        parallelism: 1
+        steps:
+            - run: echo $CIRCLE_PROJECT_USERNAME
+            - run: |
+                if [ "$CIRCLE_PROJECT_USERNAME" = "huggingface" ]; then
+                    exit 0
+                else
+                    exit -1
```

Comment:
Maybe echo something here so that when reading the log, we understand what failed?

Reply:
Oh, I see, more clear message here.

Reply:
It's now looks like (if error occurs)

```
The CI is running under the huggingface personal account. Please following https://support.circleci.com/hc/en-us/articles/360008097173-Troubleshooting-why-pull-requests-are-not-triggering-jobs-on-my-organization- to fix it.

Exited with code exit status 255
CircleCI received exit code 255
```